services/site/tailwind.config.js

```diff
@@ -82,6 +82,21 @@ module.exports = {
                 content: "none !important",
               },
             },
+            a: {
```

Comment:
I'm not happy with this. We have several occurrences of in-text links on the website (About, press kit, home page, ...). They all have the same styling. BUT they have the text and border color "light". The instruction text is the only occurrence with grey text and border. 

So maybe we can 
1. create the basic styling with the light color, 
2. remove all the unnecessary classes from the links we have on the website 
3. and make an extra styling for the link in the instructions.

Reply:
> 1. create the basic styling with the light color,
> 2. remove all the unnecessary classes from the links we have on the website

Yes, we can do that and I would support having a common styling for all links on the site. However, this is out of scope for this PR.

> 3. and make an extra styling for the link in the instructions.

That's what I'm doing in this PR